ledger-core-bitcoin/src/bitcoin/Bitcoin.cpp

```diff
@@ -0,0 +1,84 @@
+#include <core/Services.hpp>
+#include <core/api/ErrorCodeCallback.hpp>
+#include <core/api/Services.hpp>
+#include <core/api/WalletStore.hpp>
+#include <core/wallet/WalletStore.hpp>
+#include <bitcoin/Bitcoin.hpp>
+#include <bitcoin/BitcoinLikeCurrencies.hpp>
+#include <bitcoin/factories/BitcoinLikeWalletFactory.hpp>
+
+namespace {
+  ledger::core::Future<ledger::core::api::ErrorCode> registerCurrenciesInto(
```

Comment:
It seems that function is used by underlying coin implementations. Can you move it somewhere in `ledger-core` folder ?

Reply:
Sure we can.

Reply:
Done.